package.json

```diff
@@ -25,6 +25,9 @@
     "prepare": "npm run copy-testharness",
     "copy-testharness": "copyfiles -u 2 wpt/resources/testharness.js testharness/"
   },
+  "engines": {
+    "node": ">= 8"
```

Comment:
I went with Node 8 instead of Node 7, since [Node 7 is already end-of-life](https://github.com/nodejs/Release#end-of-life-releases).